tests/ecc/test_backends.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from evm.ecc.backends.pure_python import PurePythonECCBackend
+from evm.ecc.backends.coincurve import CoinCurveECCBackend
+from evm.utils.ecdsa import decode_signature
+from evm.utils.secp256k1 import (
+    decode_public_key,
+    encode_raw_public_key,
+)
+
+from tests.ecdsa_fixtures import (
+    MSG,
+    MSGHASH,
+    SECRETS,
+)
+
+
+backends = [PurePythonECCBackend()]
+try:
+    backends.append(CoinCurveECCBackend())
+except ImportError:
+    pass
```

Comment:
This worries me as it could lead to a misconfiguration in CI leaving the coincurve backend untested because it simply gets skipped.

In the same vein, I dislike having `coincurve` be a development dependency because I want to be able to test the library **without** it installed to know that it isn't required.  Not sure what the best approach here is.